--- a/src/osdep/file.c
+++ b/src/osdep/file.c
@@ -33,7 +33,7 @@
 #include <err.h>
 
 #include "osdep.h"
-#include "pcap.h"
+#include "../pcap.h"
 #include "radiotap/radiotap_iter.h"
 
 struct priv_file {
--- a/src/osdep/linux.c
+++ b/src/osdep/linux.c
@@ -62,7 +62,7 @@
          * - since linux does not include it in userspace headers
          */
 #include "osdep.h"
-#include "pcap.h"
+#include "../pcap.h"
 #include "crctable_osdep.h"
 #include "common.h"
 #include "byteorder.h"
